third_party/nvfuser/test/test_gpu_combined_inner_outer_reduction.cpp

```diff
@@ -583,4 +583,126 @@ TEST_F(NVFuserTest, FusionCombinedReductionMultiPerBlock_CUDA) {
       __FILE__);
 }
 
+TEST_F(NVFuserTest, FusionCombinedReductionLayerNormFused_CUDA) {
+  std::unique_ptr<Fusion> fusion_ptr = std::make_unique<Fusion>();
+  Fusion& fusion = *fusion_ptr.get();
+  FusionGuard fg(&fusion);
+  DataType dtype = DataType::Half;
+  std::vector<int64_t> batch_shape{2048};
+  std::vector<int64_t> norm_shape{768};
+  std::vector<int64_t> other_input_shape{2048, 1024};
+  std::vector<int64_t> input_shape(batch_shape);
+  std::copy(
+      norm_shape.begin(), norm_shape.end(), std::back_inserter(input_shape));
+
+  const size_t kM = input_shape.size();
+  const size_t kN = norm_shape.size();
+  const size_t kOuterNumDims = kM - kN;
+  std::vector<int64_t> outer_shape;
+  for (const auto idx : c10::irange(kOuterNumDims)) {
+    outer_shape.push_back(input_shape[idx]);
+  }
+  for (const auto idx : c10::irange(kOuterNumDims, kM)) {
+    outer_shape.push_back(1 + idx - idx);
```

Comment:
Why not just `push_back(1)`?

Reply:
avoid unused variable error. added comments.